cx/slack/server.js

```diff
@@ -77,13 +58,50 @@ async function detectIntentResponse(slackRequest) {
         sessionId
     );
     console.info(sessionPath);
-    
+
     request = slackToDetectIntent(slackRequest, sessionPath);
     const [response] = await client.detectIntent(request);
 
     return response;
 };
 
+async function convertToSlackMessage(responses, channel_id) {
+    let replies = [];
+
+    for(let response of responses.queryResult.responseMessages) {
+        let reply;
+
+        switch(true) {
+            case response.hasOwnProperty('text'): {
+                reply = {
+                    channel: channel_id,
+                    text: response.text.text.join()
+                };  
+            }
+            break;
+ 
+            /**
+             * For image responses the message format can be found
+             * at https://api.slack.com/reference/block-kit/block-elements#image
+             * For button responses the message format can be found at
+             * https://api.slack.com/legacy/message-buttons
```

Comment:
Probably also update the reference link here to use the up-to-date one.

Reply:
Updated the reference link